### DISCUSSION TITLE
说说 | YuZhangWang的领域

### DISCUSSION THREAD

**Discussion by giscus** [YuZhangWang/YuZhangWang.github.io#17] [Announcements]
# 说说 | YuZhangWang的领域

PLUSULTRA!

https://yuzhang.wang/natter/

**Comment by YuZhangWang** [comment]
已经没事了。要问为什么？因为我来了！

**Comment by YuZhangWang** [comment]
所谓御世制人，就是你可以参透整个世界，却永远无法了解人性。

**Comment by YuZhangWang** [comment]
人要和自己达成和解

**Comment by YuZhangWang** [comment]
有道是，谋事在人成事在天。希望一切如你所愿吧。前尘旧事一并了结，到时你在九泉之下安息，我功成身退。

**Comment by YuZhangWang** [comment]
桃李花开日，梧桐落叶时。欲遂长生志，但求千金方。

**Comment by YuZhangWang** [comment]
“到底是什么，让你从一个爱笑，眼里带光的人，变成了如今，波澜不惊，沉默不语的样子”

**Comment by YuZhangWang** [comment]
只有死鱼才会随波逐流。

**Comment by YuZhangWang** [comment]
尊严只在剑锋之上，真理只在大炮射程之内

**Comment by YuZhangWang** [comment]
郭德纲说过，黑你的人，最清楚你有多白。

**Comment by YuZhangWang** [comment]
眼看他起朱楼，眼看他宴宾客，眼看他楼塌了。

**Comment by YuZhangWang** [comment]
“我辞别了山，度过了江，划着一条小船，独自驶向人间的海洋。海风吹打我的面，海浪拍打我的船，却不曾淹没过我。”

**Comment by YuZhangWang** [comment]
你们笑我，无非是因为我头上的辫子。我头上的辫子是有形的，想剪随时可以剪掉，然而，诸位心里的辫子，就不是那么能剪掉的了。

**Comment by YuZhangWang** [comment]
苦练二十年屠龙术，结果发现现在恶龙胖成了条虫

**Comment by YuZhangWang** [comment]
喂！三点多了！饮茶先啊饮茶先，做做卵啊做，做那么多都冇用的！

**Comment by YuZhangWang** [comment]
“将军们的妻子怎么都又老又丑？要是我，肯定能当个出色的将军夫人。”

“想当将军夫人，就得先嫁给中尉，跟他在边境上、森林里、沙漠上，过上20年。”

**Comment by YuZhangWang** [comment]
难为知己难为敌

**Comment by YuZhangWang** [comment]
知道恐龙为什么灭绝吗？因为它前肢过短，无法为这种惊为天人的作品而鼓掌

**Comment by YuZhangWang** [comment]
We live in a twilight world.

And there are no friends at dusk.

**Comment by YuZhangWang** [comment]
傲上而不辱下 欺强而不凌弱

**Comment by YuZhangWang** [comment]
当然日本人也讲究礼貌，我个人认为他们的礼貌是一种没有芬芳的花儿。

**Comment by YuZhangWang** [comment]
江山父老能容我不使人间造孽钱

**Comment by YuZhangWang** [comment]
那一夜，我也曾梦到百万雄兵

**Comment by YuZhangWang** [comment]
我学文，是想坐下来和你讲道理。

我学武，是想让你坐下来听我讲道理。

**Comment by YuZhangWang** [comment]
有特别才能的人，总会在某方面显得怪怪的，显得特别不合群，格格不入。可是这些怪异处你必须忽略，因为正是这种怪异才成全了这类人才。

**Comment by YuZhangWang** [comment]
志不强者智不达 言不信者行不果

**Comment by YuZhangWang** [comment]
我看着你，你还给我耍心眼。

**Comment by YuZhangWang** [comment]
班级群里的正常人：不说话的人

**Comment by YuZhangWang** [comment]
逝去的猫猫会踏着彩虹桥来到乌托邦变成一只只模因猫

**Comment by YuZhangWang** [comment]
大事上犯糊涂，小事上耍聪明，这种人迟早得出事

**Comment by YuZhangWang** [comment]
我做事的时候跟我说，你想的太简单了；我有事的时候跟我讲，你想的太复杂了。

**Comment by YuZhangWang** [comment]
人就跟电脑一样，不是说你配置高，性能就一定好，还得看散热，如果你电脑动不动就很热，那再强的性能也发挥不出来。人也是这样，如果你遇事就慌乱，那再聪明的人也会手足无措。

**Comment by YuZhangWang** [comment]
时势变人心自然会变，就像地上的青草，也会顺应天时生长一样

**Comment by YuZhangWang** [comment]
To pursue transcendence means to accept loneliness.

**Comment by YuZhangWang** [comment]
![](https://gcore.jsdelivr.net/gh/YuZhangWang/Creative-pictures02@master/img/202306220033980.png)

**Comment by YuZhangWang** [comment]
“漠河舞厅这首歌的故事原型：一位名叫张德全的老人，在舞厅中央独自跳舞。三十多年前，他妻子在火灾中不幸离世。从此他未再婚，他与妻子无子女。妻子生前喜欢跳舞，他和她以前常在亮着灯的旧仓库里一起跳舞。这是不得已的中途散场。更多的人，是在爱的跋山涉水中半途而废。”

**Comment by YuZhangWang** [comment]
两岸猿声啼不住轻舟已过万重山

**Comment by YuZhangWang** [comment]
人生在世，注定要受许多委屈。而一个人越是成功，他所遭受的委屈也越多。

**Comment by YuZhangWang** [comment]
Putty本无树，MinGW亦非台

**Comment by YuZhangWang** [comment]
寄蜉蝣于天地，渺沧海之一粟

**Comment by YuZhangWang** [comment]
放下一朵曼陀罗

祭奠一个曾经的我

**Comment by YuZhangWang** [comment]
晓峰吹乱散发

残月远照天涯

昔日相约柳下

唯见孤影寒鸦

**Comment by YuZhangWang** [comment]
某一天我打开了地下室的门

恍惚间我看到另一个我就静静地坐在那

我没有疑惑没有迟疑，是我把他锁在了这里

**Comment by YuZhangWang** [comment]
最优秀的学生既聪明又努力，比较优秀的学生努力不聪明，一般的学生聪明不努力，拉胯的学生既不聪明又不努力。

**Comment by YuZhangWang** [comment]
莫欺少年穷

**Comment by YuZhangWang** [comment]
“好人可以通过自己的德行来吸引同道，而流氓只能寻找同类。”

“强的最小单位是贯彻自己的意志。”

**Comment by YuZhangWang** [comment]
我不欺负别人，别人也别想欺负我

如果有人硬要把我逼成对手，那么我一定会做一个合格的对手

**Comment by YuZhangWang** [comment]
不要遇见一点点挫折委屈，就轻易改变自己做人的准则和政治信仰

**Comment by YuZhangWang** [comment]
Neither your parents nor your children, only your friends can touch your private.

**Comment by YuZhangWang** [comment]
对于欠缺天分的学生而言，你的内心一定是牺牲了什么珍贵的东西，才获得了向上走的动力

**Comment by YuZhangWang** [comment]
用你上半辈子 去经历别人没有经历过的

用你下半辈子 去经历别人正在经历的

**Comment by YuZhangWang** [comment]
一时的失利，总比一时的胜利好得多

**Comment by YuZhangWang** [comment]
过去的事情，就让它过去好了

省得因为你的过去，而耽误了你的未来

**Comment by YuZhangWang** [comment]
当你想要某种东西时，整个宇宙会合力助你实现愿望。——《牧羊少年奇幻之旅》

**Comment by YuZhangWang** [comment]
PLUSULTRA！

**Comment by YuZhangWang** [comment]
有两个成语，一个叫做衣锦还乡，一个叫做锦衣夜行。

衣锦还乡意思是，古时指做官后，穿了锦绣的衣服，回到故乡向亲友夸耀。

锦衣夜行意思是，穿着华美的衣服在黑暗的夜晚走路，表示虽然事业上有所成就，却没有回乡接受亲友的称誉。

对于很多有了出头之日的人来说，锦衣夜行其实很难，不炫出来，特别难受。

**Comment by YuZhangWang** [comment]
cyber thimble

**Comment by YuZhangWang** [comment]
人与人智慧的差异是很小的，经历本身也是一种文化

**Comment by YuZhangWang** [comment]
不宜妄自菲薄，引喻失义

亦宜自谋，以咨诹善道，察纳雅言

**Comment by YuZhangWang** [comment]
Doing things right or doing right things?

Doing right things!!!

**Comment by YuZhangWang** [comment]
人生不如意十有八九 能与人言不过二三

**Comment by YuZhangWang** [comment]
小隐隐于野 大隐隐于市

**Comment by YuZhangWang** [comment]
我本将心向明月 奈何明月照沟渠

**Comment by YuZhangWang** [comment]
万法皆空 因果不空

**Comment by YuZhangWang** [comment]
杀人放火金腰带，修桥补路无尸骸

**Comment by YuZhangWang** [comment]
掉在水里你不会被淹死，呆在水里你才会淹死，你只有游不停地往前游，那些从一开始就选择放弃的人他不会失败，因为他们一开始就失败了，失败并不可怕，害怕失败才真正可怕，我们只有从失败中寻找胜利，在绝望中寻求希望。

**Comment by YuZhangWang** [comment]
所以永远不要相信一个人的成功是运气好，更加不要轻易觉得自己比领导更聪明！

**Comment by YuZhangWang** [comment]
![](https://gcore.jsdelivr.net/gh/YuZhangWang/Creative-pictures02@master/img/202306220050260.png)

**Comment by YuZhangWang** [comment]
没有什么东西是不付出就可以轻易得到的，即使你得到了，也只是暂时的，因为它本不属于你。但是，通过你自己努力之后得到的，它一定不属于别人！

**Comment by YuZhangWang** [comment]
弗知而言为不智，知而不言为不忠

**Comment by YuZhangWang** [comment]
使我有洛阳两顷田，安能配六国相印

**Comment by YuZhangWang** [comment]
告诉自己，难受是正常的，多安排一些事情让自己忙起来，哪怕痛苦也慢慢去经历。只是我们是积极接受痛苦，边走边痛苦，而不是停在原地痛苦。

**Comment by YuZhangWang** [comment]
我们弱小的时候，列强们疯狂地掠夺残害我们；我们强大的时候，列强们就疯狂的污蔑抹黑欺骗我们好让我们变弱小；时间会变，人不会变。

**Comment by YuZhangWang** [comment]
你的思维需要比你的问题的级别高才能解决该问题。

**Comment by YuZhangWang** [comment]
用自己认知里的东西去要求别人，纠正别人的行为来证明自己，本身就是有失教养的表现。

**Comment by YuZhangWang** [comment]
我是值得被爱的。

"我不被爱，不是我的错，是别人错了。"

**Comment by YuZhangWang** [comment]
“读过多少书，就敢说这个世道‘就是这样的’，见过多少人，就敢说男人女人‘都是这般德行’？你亲眼见过多少太平和苦难，就敢断言他人的善恶?”

**Comment by YuZhangWang** [comment]
我们当初之所以达到了今天的高度，靠的正是放低姿态，不放过任何一个知识点，和提高自己的机会。我们一切行动力的来源，正是来源于内心对目标的渴望和追求。

**Comment by YuZhangWang** [comment]
Be careful what you tell people.

A friend today could be an enemy tomorrow.

**Comment by YuZhangWang** [comment]
如果今天你的起点依然不高，如果今天你得到的生活不是你想要的，如果今天你所处的朋友圈的层次和水平也不是你向往的，请你选择努力，努力使自己发光，就不惧怕黑暗。当你光芒四射时，你会在黑暗中照亮自己和别人。

**Comment by YuZhangWang** [comment]
以深深的谦虚和忍耐去期待一个新的豁然贯通的时刻。

**Comment by YuZhangWang** [comment]
把理想丢给野狗，把良心留给自己，再往里面塞一把尺子，量量人心，量量世间万物，量出来谁家期货好就买谁家的，华为好就买华为，苹果好就买苹果。如果侥幸能够活下去，学一学屠龙术，会去找已经变成恶犬的野狗，用学到的屠龙术和恶犬斗一斗。如果侥幸能够斗赢，刨开它的肚子看看自己的理想有没有烂掉，如果还没让完全就塞回自己心里。

**Comment by YuZhangWang** [comment]
你们知道他们在说谎，

他们知道你们知道他们在说谎，

你们知道他们知道你们知道他们在说谎。

但是，他们还在说谎！

**Comment by YuZhangWang** [comment]
当你决定要做一件事情，不要告诉任何人，默默地把它做完， 你要做一个不动声色的大人了。不准情绪化，不准偷偷想念，不准回头看。去过自己另外的生活。——村上春树

**Comment by YuZhangWang** [comment]
换个环境，换个角度，你会发现不一样的世界

**Comment by YuZhangWang** [comment]
当你的才华还撑不起你的野心的时候，那你就应该静下心来好好学习

**Comment by YuZhangWang** [comment]
你遭受了痛苦，你也不要向人诉说，以求同情，因为一个有独特性的人，连他的痛苦都是独特的，深刻的，不易被人了解，别人的同情只会解除你的痛苦的个人性，使之降低为平庸的烦恼，同时也就使你的人格遭到贬值。

**Comment by YuZhangWang** [comment]
用理想要求自己，用世俗之见宽以待人

**Comment by YuZhangWang** [comment]
求仙问佛不如自己做主 诗佛圣经不如本事在身

**Comment by YuZhangWang** [comment]
盖有非常之功 必待非常之人

**Comment by YuZhangWang** [comment]
超脱因果，不是不沾因果，而是沾而不染，只存己念；太上忘情，也不是无情无欲，而是心无挂碍，唯修自身。

**Comment by YuZhangWang** [comment]
最近WPS会员和BiliBili大会员有一个联合会员的活动，一份钱买两家会员，一冲动就在B站买了一个联合会员，但是由于WPS更新了会员体系，于是又把旧超级会员免费兑换到新超级会员了，没想到兑换完成之后原本392天变成了421天，还多出来一个月，又把攒下来的一万多积分兑换了一年外加几个月的新体系超级会员，这些加一块一直延续到2026年。然后看到WPS也有和BiliBili的联合会员，价格还便宜许多，于是又剁手买了一年，导致在WPS在新会员系统下的超级会员来到了2027年，而B站大会员也到了2026年。说实话挺好的，B站没一天不用的，大会员用着也挺爽，WPS这么多空间，多设备超级方便，还有额外许多文档图片处理工具，这个超级会员买的真的值:sunglasses::sunglasses::sunglasses:

<table>
	<tr>
    	<td>
		<img src="https://github.com/YuZhangWang/YuZhangWang.github.io/assets/65295250/a8fb54bc-e33c-4f5d-a333-553c6ff9c536" alt="">
         <td>
<img src="https://github.com/YuZhangWang/YuZhangWang.github.io/assets/65295250/bbc1b096-fe50-4a84-901a-74b812df0789" alt="">
     </tr>
</table>

**Comment by YuZhangWang** [comment]
4.0枫丹肝完了，坏消息是少了5个宝箱，14个水之印，不想再找了，好消息目前攒到了120抽，钟离预订

**Comment by YuZhangWang** [comment]
Notein挺好用的

![Screenshot_2023-08-29-20-55-31-541_com.orion.notein.png](https://github.com/YuZhangWang/YuZhangWang.github.io/assets/65295250/12e1e750-8d17-4eb6-a59b-8898a9d7da73)

**Comment by YuZhangWang** [comment]
乡下人去的地方少🌝🌝🌝

![[神奇海螺][中国制霸].png](https://github.com/YuZhangWang/YuZhangWang.github.io/assets/65295250/7cdf43df-e889-4b95-a97a-0bf0d6853144)

**Comment by YuZhangWang** [comment]
![77117115.jpg](https://github.com/YuZhangWang/YuZhangWang.github.io/assets/65295250/c53683cf-cc03-4971-b5f6-6bb40765c9f9)

**Comment by YuZhangWang** [comment]
王国之泪是真的好玩啊🥳🥳🥳
![0100f2c0115b6000_2023-08-25_19-53-23-285.png](https://github.com/YuZhangWang/YuZhangWang.github.io/assets/65295250/82eb09b8-f08a-4dd3-b2c6-e969d66cbe90)

**Comment by YuZhangWang** [comment]
之前攒了几篇博客没写，现在开始写，却发现，妈的真的好多内容要写啊😦😦😦

**Comment by YuZhangWang** [comment]
你说这期深渊难吧，我有史以来第一次33颗星，练度算是到了一个阶段；你说简单吧，我打了快有40多次才把11-1的3颗星拿到手
12层第1次打的时候，一看有4个初生就把永冻队放下半了，结果打到12-3下半，发现boss是冰盾，不得已，只能把国家队放在下半，这才过了12-3
对我来说，4.0整个大版本，增加的boss和怪应该不是问题，问题就在于，类似于镇守挑战一样的特殊深渊关卡，太恶心了，这最能卡我脖子

**Comment by YuZhangWang** [comment]
放毒
![IMG_20230901_222537.jpg](https://github.com/YuZhangWang/YuZhangWang.github.io/assets/65295250/f8a3a79c-64d6-4189-bced-838b07139bbf)

**Comment by YuZhangWang** [comment]
今天每日委托，把惊喜大礼最后一次刷出来了，西风佑我这个成就也到拿手了，接下来蒙德所有隐藏成就，就剩下最后一次的过量的思念了，下一轮刷到就可以出蒙德辣！

**Comment by YuZhangWang** [comment]
程序员代码规范真该好好加强了，今天一个小伙伴想来互换友链，结果Pr一堆错[YuZhangWang/Blog-Friend-Link/pull#1](https://github.com/YuZhangWang/Blog-Friend-Link/pull/1)，最后看不下去了，我上手改了算了

就从他的Pr中可以读出很多信息，提交完Pr就把Fork的仓库删除了，这是有多厌恶Fork？不要说合并，我连审查都没有审查，就删库跑路了，这也太不尊重人了

他把自己的友链直接添加到文件最后面，看也不看最后面是什么，我这里最后面是失联的友链，这下他直接成了失联人士了

然后括号问题，他提交了两次Pr才删库的，第一次就把原来有的括号删掉了，第二次觉得不对劲又自己乱改，其实很反映问题，这样自以为是地改来改去，难怪会有出现💩山代码这样的问题

**Comment by YuZhangWang** [comment]
两篇巨多的调查研究博客肝完了，一篇1w5k，一篇7k，清单列表-2

![IMG_20230904_010553.jpg](https://github.com/YuZhangWang/YuZhangWang.github.io/assets/65295250/f4ec49da-eba0-4183-ba36-cabeefcfb33b)

**Comment by YuZhangWang** [comment]
一看到这身制服的时候，我也感到别扭，后来直到，看到应该脱下制服那个笑料，我才知道诺兰是故意安排这一出。现在看完了，再想一下，应该是墨菲的演技，浑身透露出来的的气息，给人他就是物理学家这样的感觉，穿上制服，就会和这股气息格格不入，这也是侧面反映演技吧

![IMG_20230904_132055.jpg](https://github.com/YuZhangWang/YuZhangWang.github.io/assets/65295250/65161c5d-37bb-4cc8-8a74-877086d87f47)

**Comment by YuZhangWang** [comment]
43抽出钟离！

**Comment by YuZhangWang** [comment]
物品获得:来自过去的钻戒💍

![IMG_20230906_113317.jpg](https://github.com/YuZhangWang/YuZhangWang.github.io/assets/65295250/9db8d688-67d2-40fd-ac9d-a524de6f5438)

**Comment by YuZhangWang** [comment]
可以蹭一篇论文了😎

**Comment by YuZhangWang** [comment]
问：为何农村地区适合建设超级计算机？
答：因为县城非常多。 ​​​

这个笑话今天正好笑到我的点上

**Comment by YuZhangWang** [comment]
世俗的成功能给人自由，不被他人说教影响的自由

**Comment by YuZhangWang** [comment]
![IMG_20230918_164300.jpg](https://github.com/YuZhangWang/YuZhangWang.github.io/assets/65295250/482b5cd9-0e5f-4405-9038-bce4049f8c9b)

**Comment by YuZhangWang** [comment]
看个电影，前面那个女的脚巨臭(先找了工作人员，男的把脚放下了，女的一直架着，所以是女的脚臭)，后来男的鼻子👃一抽一抽开始闻到臭味了，然后女的和他无论讲什么话，男的都开始嗯，嗯，嗯地敷衍她了
![IMG_20230919_130411.jpg](https://github.com/YuZhangWang/YuZhangWang.github.io/assets/65295250/9c03e7d7-7abc-47fd-9243-70b9b788d591)

**Comment by YuZhangWang** [comment]
想走的路不好走，想做人不好做。都说身不由己，这不废话吗？己不由心，身又岂能由己。

**Comment by YuZhangWang** [comment]
今天我决定将这条转发的动态置顶。第一次看这个视频是一个月前了吧，然后第二次看我转发了并且说了“好”，今天我再次深深思考了这条视频背后的科技与人文。我那么努力敲着一行行代码，学着一个个的知识点，是为了什么呢？这个视频给了我答案，我这样做的最终目的，是为了让更多和我一样平凡以及不如我的人的生活，过得更加便利、更加美好、更加体面。这才是科学与技术的目的所在，这才是我所追求的科技的人文情怀。

- [可是朋友，小米在乎](https://www.bilibili.com/video/BV1Fu4y1Q7qG/)

![62a2624d8d2dee1fae75ff48f412011c081ac0f5](https://github.com/YuZhangWang/YuZhangWang.github.io/assets/65295250/47ff563b-1ece-48bd-8730-b1f6ad590752)

**Comment by YuZhangWang** [comment]
![IMG_20230922_181302.jpg](https://github.com/YuZhangWang/YuZhangWang.github.io/assets/65295250/92fe62e2-ebd5-4bfa-9cce-68fbf80ff120)

**Comment by YuZhangWang** [comment]
尤雨溪：你报我身份证得了
![mmexport1696767553072.png](https://github.com/YuZhangWang/YuZhangWang.github.io/assets/65295250/e42390dc-adb3-4879-a6ec-d0c2e4fb9390)

**Comment by YuZhangWang** [comment]
出自任正非的三篇最新谈话

![Screenshot_2023-10-13-19-27-58-858_com.tencent.mm-edit.jpg](https://github.com/YuZhangWang/YuZhangWang.github.io/assets/65295250/0fe300c0-7d0d-43eb-9cb8-cb1a391b3989)

**Comment by YuZhangWang** [comment]
家里加了个二氧化碳浓度检测仪，窗户加了个开窗器，这样可以在浓度较高时开窗通风，省去手动开窗的麻烦![image](https://github.com/user-attachments/assets/ee4ef401-0fe0-455c-98e3-5587ac6eb60b)
